data/src/main/scala/ch.epfl.scala.index.data/project/ProjectForm.scala

```diff
@@ -30,6 +37,18 @@ case class ProjectForm(
         )
       }
 
+    val oldBeginnerIssueLabel = project.github.flatMap(_.beginnerIssuesLabel)
```

Comment:
I put it there so that when the user sets the beginner issues label in the edit project page, this code will be run so that it can grab the beginner issues right away. 

If this code was in the Guthub step, the beginner issues wouldn't get downloaded until the next time scaladex is re-indexed so the user would have to wait until then for their project to show up in the front page panel, contributing search results, ...

The next thing I want to add is to get the beginner issues in the client when the user sets the beginner issues label in the edit project page (similar to how you added updating the readme in the client), so that the user can then select which specific issues will be displayed for their project. This will get rid of the above code completely.